src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,158 @@
+// React Imports
+import React from 'react';
+import { Link } from 'react-router-dom';
+// Solid Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import AccountCircle from '@mui/icons-material/AccountCircle';
+import AppBar from '@mui/material/AppBar';
+import Badge from '@mui/material/Badge';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+
+/**
+ * NavBar Component - Component that generates NavBar section for PASS
+ *
+ * @memberof GlobalComponents
+ * @name NavBar
+ */
+
+const NavBar = () => {
+  const { session } = useSession();
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const [anchorElNotifications, setAnchorElNotifications] = React.useState(null);
+  const [openMenu, setOpenMenu] = React.useState(false);
+
+  const menuId = 'primary-search-account-menu';
+
+  const handleOpenMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+    setOpenMenu(true);
+  };
+
+  const handleNotificationsMenu = (event) => {
+    setAnchorElNotifications(event.currentTarget);
+  };
+
+  const handleNotificationsClose = () => {
+    setAnchorElNotifications(null);
+  };
+
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar position="static" color="primary">
+        <Toolbar>
+          {session.info.isLoggedIn ? (
+            <>
+              <Typography variant="h4" noWrap component="div" mr="10px">
+                PASS
+              </Typography>
+              <>
+                <Link to="/PASS/home/">
+                  <Button
```

Comment:
For the buttons for Home (Line 60 to 66) and Forms (Lines 69 to 75), I don't think we need them. The Links work just fine.

![Screen Shot 2023-05-09 at 14 35 54](https://github.com/codeforpdx/PASS/assets/14917816/a88b650c-eaca-4883-aceb-99b45be995d6)

Reply:
Also I've just noticed, is there a way to remove that blue(?)-ish border from the header? If not, we should have it like with the Login component where they're simply using a thin box shadow.

Reply:
If you mean removing the MUI Buttons, this is the result:
![2023-05-09](https://github.com/codeforpdx/PASS/assets/83156697/29dead83-b58e-4951-bc8a-208b964d5cfe)

That border is from style.css lines 10-12:

`
header {
  border-bottom: #5795b9 solid;
  padding-left: 0.5rem;
}
`

Should they be deleted?

Reply:
Found the blue `border`... took me a minute.

@andycwilliams looks like the `AppBar` MUI component renders a `<header>` element with a border-bottom and that blue color.  I could only get rid of it with an `sx` on the `AppBar` -> `sx={{borderBottom: 'none'}}` worked for me.
Take a look.   It's probably part of the default MUI theme would be my guess. I never noticed it before.

Good catch KaHung. random.

Reply:
@xscottxbrownx Have you looked at the style.css file? Lines 10-12 alter the header element. Removing the styling from that gets rid of the border. That isn't part of MUI.

Reply:
That `<header>` also has a left padding of `0.5rem` no clue why, but seems weird... maybe see what you think about getting rid of that too?
<img width="1241" alt="Screenshot 2023-05-09 at 5 39 06 PM" src="https://github.com/codeforpdx/PASS/assets/71395931/0fc543c2-a278-46ef-962b-10642062d57d">

Reply:
> @xscottxbrownx Have you looked at the `style.css` file? Lines 10-12 alter the header element. Removing the styling from that gets rid of the border. That isn't part of MUI.
> 
> We also should coordinate on how much of the `style.css` file we want to keep there and how much we want to try and add to the `theme.js` file.

Oh perfect!  

I keep forgetting that file exists honestly.  We should probably wipe anything that we can from there I would think...  I'm for keep it all MUI if possible. Just have one source for styling.  But not sure if we'll run into any limitations or not. We'll find out.

Reply:
I haven't worked on the 'style.css' file but would be fine removing some of it.

Also, as I've been working on another component, the Upload Document one, the `.panel` styling clashes with MUI so far. So it would help to know if we want to keep that or if it's just a placeholder.

Reply:
I haven't tried it yet, but we can probably migrate everything from `style.css` to `theme.js`.

Some documentation:
https://mui.com/material-ui/react-typography/

Scott, maybe you could tackle that next? I'm still working on a couple of components.

Reply:
> If you mean removing the MUI Buttons, this is the result: ![2023-05-09](https://user-images.githubusercontent.com/83156697/237247099-29dead83-b58e-4951-bc8a-208b964d5cfe.png)
> 
> That border is from style.css lines 10-12:
> 
> `header { border-bottom: #5795b9 solid; padding-left: 0.5rem; }`
> 
> Should they be deleted?

Personally I like the text links better, but maybe use the `Tabs` MUI component?  Especially colored tabs using the `theme.palette.tertiary.main color` ?? That's my thought/idea.  https://mui.com/material-ui/react-tabs/#colored-tab

Reply:
> If you mean removing the MUI Buttons, this is the result:
> ![2023-05-09](https://github.com/codeforpdx/PASS/assets/83156697/29dead83-b58e-4951-bc8a-208b964d5cfe)

Can we not just place them in a Container, Box, or Grid component and set a gap between items? We have something similar before, but with a flex box.

> That border is from style.css lines 10-12:
> 
> `
> header {
>   border-bottom: #5795b9 solid;
>   padding-left: 0.5rem;
> }
> `
> 
> Should they be deleted?

Yes, I don't see a purpose to a bottom border.

Reply:
MUI has a specific tab component, which I'm currently trying to make work with the React Links. But I can try other types of components as well.

Reply:
> MUI has a specific tab component, which I'm currently trying to make work with the React Links. But I can try other types of components as well.

If you can get those to work, I think they looked the best out of the text link options.  And try the `theme.palette.tertiary.main` color - that golden yellow. Think it'll look good! and the header and footer would match better too.

I was looking at them earlier, is it something like:
```js
<Tabs all the attributes>
   <Tab label="link name">
      <Link all the stuff />
   </Tab>
</Tabs>
```
???